### PR TITLE
Release 0.5.3

### PR DESCRIPTION
**Pull request by kohr-h**
PR for the release. Should go in quickly.

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -14,14 +14,14 @@ build:
...
-        - python 2.7|>=3.4
+        - python
         - setuptools
         - nomkl # [not win]
         - future >=0.14
         - numpy >=1.9
         - scipy >=0.14
     run:
-        - python 2.7|>=3.4
+        - python
```
Comment (adler-j) [inline conda/meta.yaml]: Why this change? I thought we required >=3.4?

File: setup.py [hunk 2]
```diff
@@ -94,14 +94,15 @@ def find_tests():
 - Objects to represent mathematical notions like vector spaces and operators, including properties as expected from mathematics (inner product, norm, operator composition, ...)
 - Convenience functionality for operators like arithmetic, composition, operator matrices etc., which satisfy the known mathematical rules.
 - Out-of-the-box support for frequently used operators like scaling, partial derivative, gradient, Fourier transform etc.
+- A versatile and pluggable library of optimization routines for smooth and non-smooth problems, such as CGLS, BFGS, Forward-Backward splitting and Douglas-Rachford splitting.
```
Comment (adler-j) [inline setup.py]: Change Forward-Backward splitting to Chambolle-Pock, since it is more famous.

File: doc/source/release_notes.rst [hunk 3]
```diff
@@ -9,6 +9,76 @@ Release Notes
...
+- Add ``CallbackPrint`` for printing of intermediate results in iterative solvers (:pull:`691`).
+- Expose Numpy ufuncs as operators in the new ``ufunc_ops`` subpackage (:pull:`576`).
+- Add ``ScalingFunctional`` and ``IdentityFunctional`` (:pull:`576`).
+- Add ``RealPart``, ``ImagPart`` and ``ComplexEmbedding`` operators (:pull:`706`).
+- Add ``PointwiseSum`` operator for vector fields (:pull:`754`).
+- Add ``LineSearchFromIterNum`` for using a pre-defined mapping from iteration number to step size (:pull:`752`).
+- Add ``axis_labels`` option to ``DiscreteLp`` for custom labels in plots (:pull:`770`).
+- Add Defrise phantom for cone beam geometry testing (:pull:`756`).
+- Add ``filter`` option to ``fbp_op`` and ``tam_danielson_window`` helper for helical cone beam geometry (:pull:`756`, :pull:`806`).
+- Add ISTA (``proximal_gradient``) and FISTA (``accelerated_proximal_gradient``) algorithms for L1 regularization (:pull:`758`).
```
Comment (adler-j) [inline doc/source/release_notes.rst]: Much more general than l1 regularization

File: doc/source/release_notes.rst [hunk 4]
```diff
@@ -9,6 +9,76 @@ Release Notes
...
+- Add ``salt_pepper_noise`` helper function (:pull:`758`).
+- Expose filtering as operator ``fbp_filter_op`` (:pull:`780`).
```
Comment (adler-j) [inline doc/source/release_notes.rst]: Expose **FBP** filtering...

File: doc/source/release_notes.rst [hunk 5]
```diff
@@ -9,6 +9,76 @@ Release Notes
...
+------------
+- New documentation on the operator derivative notion in ODL (:pull:`668`).
+- Add largescale tests for the convex conjugates of functionals (:pull:`744`).
+- Add ``domain`` parameter to ``LinDeformFixedTempl`` for better extensibility (:pull:`748`).
+- Add example for sparse tomography with TV regularization using the Douglas-Rachford solver (:pull:`746`).
+- Add support for 1/r^2 scaling in cone beam backprojection with ASTRA 1.8 using a helper function for rescaling (:pull:`749`).
+- Improve performance of operator scaling in certain cases (:pull:`576`).
+- Add documentation on testing in ODL (:pull:`704`).
+- Replace occurrences of ``numpy.matrix`` objects (:pull:`778`).
+- Allow creation of phantoms in a space with degenerate shape (:pull:`777`).
```
Comment (adler-j) [inline doc/source/release_notes.rst]: This was a bug imo

File: conda/meta.yaml (hunk 1 above)
Comment (kohr-h) [inline conda/meta.yaml]: The problem is that it pulls in 3.6 during build even if I specify `--python 3.5`. Not sure if it's a conda bug but with this setup the correct version is pulled in.

File: setup.py (hunk 2 above)
Comment (kohr-h) [inline setup.py]: OK

**Comment by kohr-h** [comment]
> Lets get the astra improvements in
> 
> #802
> 
> Especially for 2d, they are simply immense and users shouldn't have to wait.

Okay, I'll do a review.

File: doc/source/release_notes.rst (hunk 5 above)
Comment (kohr-h) [inline doc/source/release_notes.rst]: Fair enough

File: doc/source/release_notes.rst (hunk 3 above)
Comment (kohr-h) [inline doc/source/release_notes.rst]: Correct

**Comment by kohr-h** [comment]
I'll wait until #825 is in, add them to the notes and then make the release.

**Comment by adler-j** [comment]
Give #822 a short look as well, it's a change to one of our main API's, and it will throw some people off if examples don't work.

**Comment by kohr-h** [comment]
> Give #822 a short look as well, it's a change to one of our main API's, and it will throw some people off if examples don't work.

Included in the notes now, along with the rest. Waiting for the merge of #826, then I make the release tag.

**Comment by kohr-h** [comment]
All good with this version?
